integration/confidential/fixtures/quay_verification/quay_policy.json

```diff
@@ -1,18 +1,18 @@
 {
-    "default": [
+  "default": [
+    {
+      "type": "insecureAcceptAnything"
+    }
+  ],
+  "transports": {
+    "docker": {
+      "quay.io/kata-containers/confidential-containers": [
         {
-            "type": "insecureAcceptAnything"
-        }
-    ],
-    "transports": {
-        "docker": {
-            "quay.io/kata-containers": [
-                {
-                    "type": "signedBy",
-                    "keyType": "GPGKeys",
-                    "keyPath": "/etc/containers/quay_verification/public.gpg"
-                }
-            ]
+          "type": "signedBy",
```

Comment:
Changes in this file except `"quay.io/kata-containers" -> "quay.io/kata-containers/confidential-containers"` seems only about alignment, do we need them?

Reply:
I agree, it is not needed.